src/main/kubectl.ts

```diff
@@ -289,7 +289,7 @@ export class Kubectl {
   }
 
   protected getDownloadMirror() {
-    const mirror = packageMirrors.get(userStore.preferences.downloadMirror)
+    const mirror = packageMirrors.get(userStore.preferences?.downloadMirror)
```

Comment:
This is weird because `preferences` object (not empty) defined directly in the class..